tests/unit-tests/test_loss.py

```diff
@@ -108,8 +109,29 @@ def test_FocalLossFunc():
     assert_almost_equal(loss.numpy(), gt_loss.numpy(), decimal=4)
 
 
+def test_ShrinkageLossFunc():
+    alpha = 10.
+    gamma = 0.2
+    loss_func = ShrinkageLossFunc(alpha, gamma)
+    logits = th.tensor([0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0])
+    labels = th.tensor([1., 1., 1., 1., 1., 1., 1., 1., 0, 0.9])
+    gt_loss = th.tensor(0.3565)
```

Comment:
We can paste the loss = \frac{l^2}{1 + \exp \left( \alpha \cdot (\gamma - l) \right)} here, about how we get the value.